pkg/remote/scanner.go

```diff
@@ -57,6 +60,12 @@ loop:
 
 func (s *Scanner) scan() ([]resource.Resource, error) {
 	for _, enumerator := range s.remoteLibrary.Enumerators() {
+		if s.filter.IsTypeIgnored(enumerator.SupportedType()) {
```

Comment:
Could `s.filter` be nil ?

Reply:
No, we pass a driftignore struct every time, if the driftignore is empty or does not exist we still pass a driftignore struct in `scan.go` but it will have empty matcher. So this should work